ruby/red-arrow/lib/arrow/slicer.rb

```diff
@@ -162,6 +162,11 @@ def select(&block)
       def reject(&block)
         RejectCondition.new(@column, block)
       end
+
+      def match_substring(substring, ignore_case: false)
+        MatchSubstringCondition.new("match_substring",
```

Comment:
How about `MatchSubstringFamilyCondition` because this is not only for `match_substring`?

Reply:
Thnaks!